portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,22 +24,60 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+/** Class used to store comment data */
```

Comment:
I would make this into its own file. No need for it to be in DataServlet.java. I also recommend making this an AutoValue class.

Reply:
Sorry these changes were supposed to be in a different branch but carried over, I've since removed them.

Reply:
Thanks, that makes sense. I was a little confused why they were added.